.github/workflows/hardhat.yml

```
@@ -12,7 +12,7 @@ jobs:
       - name: Use Node.js 18
         uses: actions/setup-node@v3
```

actionlint (1.7.10)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)